module/README.md

```diff
@@ -0,0 +1,46 @@
+## Early MVP
+Happy path implementations
+
+### Oracle
+
+#### Assumptions
+- An orchestrator may want to submit multiple claims with a msg (withdrawal batch update + MultiSig Set update )
+- Nonces are not unique without a context (withdrawal nonce and MultiSig Set update can have same nonce (=height))
+- A nonce is unique in it's context and never reused
+- Multiple claims by an orchestrator for the same ETH event are forbidden
+- We know the ETH event types beforehand (and handle them as ClaimTypes) 
+- For an **observation** status in Attestation the power AND count thresholds must be exceeded
+- Fraction type allows higher precision math than %. For example with 2/3
+
+A good start to follow the process would be the `x/peggy/handler_test.go` file
+
+### Outgoing TX Pool
+#### Features
+* Unique denominator for peggy vouchers in cosmos (🚧  cut to 15 chars and without a separator due to sdk limitations in v0.38.4)
```

Comment:
could we be more clear about what cases are supported with batches? We can make and observe submission of a batch?

What about the valset status. Does the oracle for valset submission allow for that validator set to be used for checking if a message is `confirmed`? Or is that not implemented yet.

Reply:
The batch implementation bundles pending TX into a batch object. No CLI/ Endpoints yet to make use of it.
For integrating valSet updates the oracle side needs to be in place before. I would link to the last "observed" MultSig update when a batch is created. Then the endpoint can load and return all the relevant data for the clients.
Happy path only :-)